TestAssets/TestProjects/AppWithLibrary/TestApp/TestApp.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" />
+  <Import Project="$(MSBuildSDKsPath)\microsoft.net.sdk\Sdk\Sdk.props" />
```

Comment:
Also, the folder name should be spelled sdk if this is going to be in a nuget package since lowercase is a convention there. Later I see it that way. Perhaps that's what's making Linux builds unhappy.